### PR TITLE
[Prod-968] debug r version

### PR DESCRIPTION
**Pull request by jdcanas**

### CODE REVIEW

File: http/src/main/resources/init-resources/azure_vm_init_script.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-set -e
+#set -e
```
Comment (LizBaldo) [inline http/src/main/resources/init-resources/azure_vm_init_script.sh]: Could you uncomment this?

Comment (jdcanas) [reply http/src/main/resources/init-resources/azure_vm_init_script.sh]: yeah, I have that locally, just waiting on this run of test

File: http/src/main/resources/init-resources/azure_vm_init_script.sh [hunk 2]
```diff
@@ -176,15 +176,6 @@ while sudo fuser /var/lib/dpkg/lock-frontend > /dev/null 2>&1
     sleep 5
   done
 
-# Install updated R version
-echo "Installing R version ${R_VERSION}"
```
Comment (LizBaldo) [inline http/src/main/resources/init-resources/azure_vm_init_script.sh:180]: Could you also remove the R_VERSION variable that is set above?

**Review by LizBaldo — APPROVED** [review]
I have a couple of comments to address, but 👍 to unblock you

**Review by lucymcnatt — APPROVED** [review]
LGTM!

**Pull request merged**